src/cascadia/TerminalSettingsEditor/Appearances.xaml

```diff
@@ -144,7 +134,7 @@
                                 Minimum="0"
                                 TickFrequency="50"
                                 TickPlacement="Outside"
-                                Value="{x:Bind Appearance.FontWeight, Converter={StaticResource FontWeightConverter}, Mode=TwoWay}" />
+                                Value="{x:Bind local:Converters.FontWeightToDouble(Appearance.FontWeight), BindBack=Appearance.SetFontWeightFromDouble, Mode=TwoWay}" />

```

Comment:
Unfortunately, this doesn't seem to work since two way binding either wants to bind to a property for both directions or have a function for both directions. Having a property for one direction (as would be the case with simply using Appearance.FontWeight.Weight) and a BindBack for the other direction does not seem to be supported.